src/types.jl

```diff
@@ -160,6 +160,21 @@ struct RGB4{T<:Fractional} <: AbstractRGB{T}
 end
 RGB4(r::T, g::T, b::T) where {T<:Fractional} = RGB4{T}(r, g, b)
 
+"""`RGB565` is Red-Green-Blue colorspace represented by two bytes (16 bits).
+
+1        |1          |        0
+5 4 3 2 1|0 9 8 7 6 5|4 3 2 1 0 
+---------|-----------|---------
+R R R R R|G G G G G G|B B B B B
+"""
+struct RGB565 <: AbstractRGB
+    value::UInt16
+end
+
+function RGB565(r, g, b)
```

Comment:
You could call this `RGB565_integer` or something, but the default constructor has to respect the 0-to-1 colorscale (essentially being the converse of `red`, `green`, `blue`). Otherwise you get into a situation where `1` has a different *mathematical* (rather than computational) meaning than `1.0`, which would be a disaster.

`RGB565` itself has to be something like

```julia
function RGB565(r::N3f5, g::N2f6, b::N3f5)
    # body. Needs to check that all inputs are between 0 and 1.
end
RGB565(r, g, b) = RGB565(N3f5(r), N2f6(g), N3f5(b))
```